rebar.config

```diff
@@ -4,7 +4,7 @@
 {deps, [
         {ibrowse, ".*",
          %% Pin here, becase 555f707 (pr #155) introduces an ipv6 bug we've not fixed
-         {git, "https://github.com/cmullaparthi/ibrowse.git", {ref, "c97136cfb61fcc6f39d4e7da47372a64f7fca04e"}}},
+         {git, "git@github.com:cmullaparthi/ibrowse.git", {ref, "c97136cfb61fcc6f39d4e7da47372a64f7fca04e"}}},
         {envy, {git, "https://github.com/markan/envy.git", {branch, "master"}}}
```

Comment:
Aren't we going to have issues with this other dependency too?

Reply:
Not by the looks of it? The issue is that the link format was wrong in the lock file, I guess I could have changed it to just use HTTPS as well. Should I go with that instead, or perhaps change envy to use git as well?